### PR TITLE
Highlight the deadline extension

### PR DESCRIPTION
**Pull request by KathyReid**
Triggered by a corridor conversation b/w Kathy and Maia

_Very minor change, but am following our established review process because I think that patterns and rhythms of behaviour establish cultural norms and expecations._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ![guiltyrobot](guilty-robot.jpg "Guilty Robot")
 
+```
+DEADLINE EXTENDED UNTIL 1st SEPTEMBER
+
+The deadline for 2020 Applications for the Masters in Applied Cybernetics has now been extended until 1st September
```
Comment (Zac-HD) [inline README.md:6]:
```suggestion
The application deadline for the 2020 Masters in Applied Cybernetics
has now been extended until 1st September.
```

Slight rephrasing, and reorganised for readability on narrow (e.g. mobile) screens.

**Review by Zac-HD — APPROVED** [review]
LGTM with a minor tweak; go ahead and merge if you're happy with it :smile:

File: README.md (hunk 1 above)
Comment (KathyReid) [reply README.md:6]: Excellent re-phrasing / re-wrapping suggestion, appreciated. 
Merging :-)

**Comment by ghjansen** [comment]
I wish I could apply twice now. 😜

**Pull request merged**